src/Form/DataTransformer/ModelToIdPropertyTransformer.php

```diff
@@ -108,12 +108,16 @@ public function reverseTransform($value)
             throw new \UnexpectedValueException(sprintf('Value should be array, %s given.', \gettype($value)));
         }
 
-        foreach ($value as $key => $id) {
-            if ('_labels' === $key) {
-                continue;
-            }
+        $ids = array_filter($value, static function ($key): bool {
+            return '_labels' !== $key;
+        }, ARRAY_FILTER_USE_KEY);
+
+        $query = $this->modelManager->createQuery($this->className);
+        $this->modelManager->addIdentifiersToQuery($this->className, $query, $ids);
+        $result = $this->modelManager->executeQuery($query);
```

Comment:
If this is already a collection, shouldn't we return it directly?

Reply:
To me `executeQuery` can execute anything, it could create, read, update and delete, so I'm not sure about adding a return type declaration. I find `ModelManagerInterface::findMultiple()` better, it's clearer what it does and makes sense to me.

Reply:
Good point. I'll try to follow this work later.
As a side note about the "query" concept, the ORM implementation of `ModelManagerInterface::createQuery()` is always creating a `SELECT` query.
If we want to be clear about that,, maybe we should use `createSelectQuery()`, `configureSelectQuery()`, etc.

Reply:
I need thought of `executeQuery` doing something else than a select.
But IMHO there is no return type cause it can be an object (when looking for one element) or an array (for multiple).

Reply:
I've added `ModelManagerInterface::findMultiple(string $class, array $ids): Doctrine\Common\Collections\Collection;`.
Then, we should implement this method in the corresponding libraries.
Please, confirm if this seems fine for you.

Reply:
There is
```
/**
 * @param string               $class
 * @param array<string, mixed> $criteria
 *
 * @return object[] all objects matching the criteria
 *
 * @phpstan-template T of object
 * @phpstan-param class-string<T> $class
 * @phpstan-return T[]
 */
public function findBy($class, array $criteria = []);
```

I think we should have consistency in the return type of `findBy` and `findMultiple`.
Both collection, or both arrays.

Reply:
>I think we should have consistency in the return type of findBy and findMultiple.
Both collection, or both arrays.

I agree. Let's use `array` as return type by now then, in order to make less complex these changes. Later we can evaluate if we should change to `Collection` or other types.
Regarding the method name, I think we could also use `findByIds()` or similar. I'm opening a room for this choice since "multiple" isn't a common term in the Doctrine `find*()` methods.

Reply:
Yes indeed, array will be easier.

I don't really like `findByIds` either.
- The `find` method is not called `findById`.
- `findByIds` seems to much related to the `findBy` method, and then I will ask myself "Why there is no `findByNames` method or similar.

A solution could also be to improve the `findBy` method to allow something like
```
$model->findBy(['id' => [3, 4, 5], 'firstName' => 'foo', 'lastName' => ['bar', 'baz'])
```

`findMultiple` (or `multipleFind` ?) isn't fully clear to me.  But seems more `find`-related.

Reply:
The issue with your proposal is that the identifier can be composed by multiple fields. IMO, that's why the magic `findBy()` method is not suitable for this case.